tests/python/gaia-ui-tests/gaiatest/tests/functional/bzlite/test_login_bzlite.py

```diff
@@ -0,0 +1,17 @@
+from gaiatest import GaiaTestCase
+from gaiatest.apps.bzlite.app import Bugzilla_lite
+
+
+class TestBugzillaLite(GaiaTestCase):
+
+    def setUp(self):
+        GaiaTestCase.setUp(self)
+        self.connect_to_local_area_network()
+
+    def test_login_BugzillaLite(self):
+        BugzillaLite = Bugzilla_lite(self.marionette)
```

Comment:
Nit: As we follow PEP-8, an instance of a class must be `instance_name` instead of `InstanceName`. Then, here, we should have:
```
bugzilla_lite = BugzillaLite(self.marionette)
```

https://www.python.org/dev/peps/pep-0008/#id38